includes/class-wc-subscriptions-cart.php

```diff
@@ -410,20 +406,13 @@ public static function calculate_subscription_totals( $total, $cart ) {
 		if ( apply_filters( 'wcs_remove_fees_from_initial_cart', $remove_fees_from_cart, $cart, $recurring_carts ) ) {
 			$cart_fees = WC()->cart->get_fees();
 
-			if ( wcs_is_woocommerce_pre( '3.2' ) ) {
-				foreach ( $cart_fees as $fee_index => $fee ) {
-					WC()->cart->fees[ $fee_index ]->amount = 0;
-					WC()->cart->fees[ $fee_index ]->tax    = 0;
-				}
-			} else {
-				foreach ( $cart_fees as $fee ) {
+			foreach ( $cart_fees as $fee ) {
```

Comment:
With the removal of the if/else nesting, the spacing here is off. 
The closing } and contents of this foreach should be brought back 1 tab.